integration/spark2/src/main/scala/org/apache/spark/sql/CarbonSession.scala

```diff
@@ -110,6 +110,14 @@ class CarbonSession(@transient val sc: SparkContext,
     )
   }
 
+  /**
+   * Return true if the specified sql statement will hit the datamap
+   */
+  def isDataMapHit(sqlStatement: String, dataMapName: String): Boolean = {
```

Comment:
Test method move to testcase

Reply:
I have added an annotation for it, I think it is useful in multiple test suite, so add it here